src/people/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: People
+---
+
+{% for person in people -%}
+<div class="person-container">
+  <div class="person-card">
+    <img src="{{ person.image }}" />
+    <div>
+      <h3>{{ person.name }}</h3>
+      {{ person.type }}<br/>
+      {{ person.location }}
+      <ul>
+        {% for name, url in person.links -%}
+        <li><a href="{{ url }}">{{ name }}</a></li>
+        {% endfor -%}
+      </ul>
+    </div>
+  </div>
+  <p><strong>Specialties:</strong> <em>{{ person.specialties }}</em></p>
+  <p><strong>Interests:</strong> <em>{{ person.interests }}</em></p>
+  <p>{{ person.bio }}</p>
+</div>
```

Comment:
The current object hierarchy is:

```
- .person-container
  - .person-card
    - img
    - div
      - h3
      - ul
        -li 
  - p
  - p 
```

Maybe something like this would reveal the intent a bit better?

```
.person
  h3.name
  img.photo alt={{ person.name }}
  main
    p.bio
    ul.links
      li
  footer
    h4 Specialties # Note: Can style both the h4 and the p to be an inline-block or inline-flex so they stay on the same line together while preserving the information hierarchy
    p
    h4 Interests
    p
```
It'd be even fancier if we conformed to the [hCard format for making the data parsable](http://microformats.org/wiki/h-card)

Reply:
I've made the markup a little bit closer to the intent while preserving the same look. It tries to strike some balance between a cleaner markup and minimize the amount of fancy bending-over-backward CSS trickery to make things look "just so" while keeping the markup strictly semantic.

There might be some simple or obvious CSS tricks that I'm missing that would make this even better, feel free to suggest if you see anything like that. That said, I am skeptical of the value of doing too much to make the markup perfectly semantic at the expense of increased CSS complexity.

Reply:
Dig it!